src/extensions/default/bramble/lib/MouseManagerRemote.js

```diff
@@ -0,0 +1,93 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, indent: 4, forin: true, maxerr: 50, regexp: true, bitwise: true */
+/* global addEventListener, sessionStorage */
+(function(transport) {
+    "use strict";
+
+    var SCROLL_KEY = "___bramble-preview-scrollTop::" + window.___brambleFilename;
+
+    function sendHighlightInfo(e, isClick) {
+        var elem = document.elementFromPoint(e.pageX - window.pageXOffset, e.pageY - window.pageYOffset);
+        if(!elem) {
+            return;
+        }
+
+        var startLine = elem.dataset.brambleStartLine;
+        if(!startLine) {
+            return;
+        }
+
+        var startCh = elem.dataset.brambleStartCh;
+        if(!startLine) {
```

Comment:
I would prefer if you initialize all the variables one after the other and then have one if statement to check all the values (looks a bit cleaner).